velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -146,7 +145,8 @@ const ::substrait::Expression& VeloxToSubstraitExprConvertor::toSubstraitExpr(
     scalarExpr->set_function_reference(functionMap_[functionName]);
 
     for (auto& arg : inputs) {
-      scalarExpr->add_args()->MergeFrom(toSubstraitExpr(arena, arg, inputType));
+      scalarExpr->add_arguments()->mutable_value()->MergeFrom(
```

Comment:
This change is part of #2383, right? Would you rebase this PR on top of #2383 so that there are 2 separate commits. First commit contains changes in #2383, second commit contains additional changes. This will make it easier to review as I'll be able to focus only on the changes in the 2nd commit.

Reply:
> Sounds good. If you want a review earlier, than, please, rebase without waiting for #2383 to be merged.

Finish rebase, and fix all comments.